app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1134,6 +1152,7 @@ BrowserWorld::StartFrame() {
     if (relayoutWidgets) {
       UpdateVisibleWidgets();
     }
+    m.handModelsRenderer->StartFrame();
```

Comment:
I wonder whether we can get rid of these knowing that we have empty implementations.

Reply:
For HandModels files, I copied the structure of ExternalBlitter, and it follows the logical life-cycle of a frame (start, draw, end/cancel). I think there is value in leaving this flow explicit, but don't feel strongly about it.

Reply:
Do you still prefer to remove them? Either way is fine for me.

Reply:
Yeah I think we can remove them because we don't have a clear use case for them. We can always add them later